src/util.cpp

```diff
@@ -1705,31 +1722,38 @@ QString GetVersionAndNameStr ( const bool bDisplayInGui )
         strVersionText += "\n *** Foundation; either version 2 of the License, or (at your option) any later version.";
         strVersionText += "\n *** There is NO WARRANTY, to the extent permitted by law.";
         strVersionText += "\n *** ";
-        strVersionText += "\n *** Using the following libraries, resources or code snippets:";
+
+        strVersionText += "\n *** " + QCoreApplication::tr ( "This app uses the following libraries, resources or code snippets:" );
         strVersionText += "\n *** ";
-        strVersionText += QString ( "\n *** Qt framework %1" ).arg ( QT_VERSION_STR );
-        strVersionText += "\n *** <https://doc.qt.io/qt-5/lgpl.html>";
+        strVersionText += "\n *** " + QCoreApplication::tr ( "Qt cross-platform application framework" ) + QString ( " %1" ).arg ( QT_VERSION_STR );
+        strVersionText += "\n *** <https://www.qt.io>";
         strVersionText += "\n *** ";
         strVersionText += "\n *** Opus Interactive Audio Codec";
         strVersionText += "\n *** <https://www.opus-codec.org>";
         strVersionText += "\n *** ";
 #if defined( _WIN32 ) && !defined( WITH_JACK )
         strVersionText += "\n *** ASIO (Audio Stream I/O) SDK";
         strVersionText += "\n *** <https://www.steinberg.net/developers>";
+        strVersionText += "\n *** ASIO is a trademark and software of Steinberg Media Technologies GmbH";
         strVersionText += "\n *** ";
 #endif
-        strVersionText += "\n *** Audio reverberation code by Perry R. Cook and Gary P. Scavone";
+#ifndef SERVER_ONLY
+        strVersionText += "\n *** " + QCoreApplication::tr ( "Audio reverberation code by Perry R. Cook and Gary P. Scavone" ) +
+                          ", 1995 - 2021, The Synthesis ToolKit in C++ (STK)";
         strVersionText += "\n *** <https://ccrma.stanford.edu/software/stk>";
         strVersionText += "\n *** ";
-        strVersionText += "\n *** Some pixmaps are from the Open Clip Art Library (OCAL)";
+#endif
+#ifndef HEADLESS
+        strVersionText += "\n *** " + QCoreApplication::tr ( "Some pixmaps are from the" ) + " Open Clip Art Library (OCAL)";
```

Comment:
I'd put the Open Clip Art library into the translation, as there might be different ways to translate it in different languages.

Reply:
No. We shouldn't translate them. However, in other languages the names might be inside a sentence due to different grammar.

Reply:
Ah, right, I get you - so we need `QCoreApplication::tr ( "Some pixmaps are from the %1" ).args ( "Open Clip Art Library (OCAL)" );`.

Reply:
That's an option

Reply:
See update.